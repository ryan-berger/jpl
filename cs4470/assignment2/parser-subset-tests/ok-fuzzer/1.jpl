read image "+" to Q
time read image "" to p1O.4e_8U_
show o9Q.()
print ""
write image J() to ""
write image 865 to "2y"
show B5n_()
write image 48 to ""
return C1V_0v_(.3268)
show 6.
assert B(), ""
