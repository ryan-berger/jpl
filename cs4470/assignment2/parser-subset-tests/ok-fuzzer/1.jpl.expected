(ReadImageCmd "+" (VarArgument Q))
(TimeCmd (ReadImageCmd "" (VarArgument p1O.4e_8U_)))
(ShowCmd (CallExpr o9Q.))
(PrintCmd "")
(WriteImageCmd (CallExpr J) "")
(WriteImageCmd (IntExpr 865) "2y")
(ShowCmd (CallExpr B5n_))
(WriteImageCmd (IntExpr 48) "")
(StmtCmd (ReturnStmt (CallExpr C1V_0v_ (FloatExpr 0))))
(ShowCmd (FloatExpr 6))
(StmtCmd (AssertStmt (CallExpr B) ""))
